### ISSUE TITLE
FireEye AX Analyzer

### ISSUE THREAD

**Issue by bullerdude** [CERT-BDF/Cortex-Analyzers#24]
###Request Type
Analyzer Request

###Work Environment
N/A

###Problem Description
Create an analyzer that will submit files to a local FireEye AX sandbox instance and retrieve the report and indicators that are generated

**Comment by saadkadhi** [comment]
Hi @bullerdude. Do you have access to such as a sandbox and if yes, would you be willing to contribute an analyzer for it should we give you all the documentation/instructions on how to do so?

**Comment by bullerdude** [comment]
Hi @saadkadhi, we are keen to develop this analyser. We will develop the code using a forked repository - https://github.com/UNIT777/Cortex-Analyzers.

**Comment by saadkadhi** [comment]
Thanks @bullerdude. Once your analyzer is ready, please submit a pull request so we integrate it to the analyzer list. Also, let us know if we can be of any help for coding the analyzer. 

Once it is dubbed ready for production, we will work on TheHive's short and long report templates. For that, we will need some sample outputs from the analyzer. Otherwise, if you feel confortable with AngularJS, please go ahead and provide short and long report templates so we can package them and share them with the community.

Thanks a heap for your help.

**Comment by bullerdude** [comment]
Hi @saadkadhi, we should have this analyser and basic report templates up in a pull request next week.

**Comment by jeromeleonard** [comment]
Hi @bullerdude, are you still working on this analyzer ?

**Comment by bullerdude** [comment]
Yes - My team should have just updated for the latest Cortex – let me check and get back to you.
Will see if we can push it to Github this week. May also seek some assistance on improving the long report template.

**Comment by BrevilleBro** [comment]
The FireEye AX analyzer has now been pushed into our repo. It currently only supports file submissions, however, we are working  #on the URL submission and will push that when ready. #78
